web/modules/custom/area_summary/src/Form/AreaSummaryForm.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Drupal\area_summary\Form;
+
+use Drupal\Component\Utility\Random;
+use Drupal\Core\Entity\ContentEntityForm;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Form controller for the area summary edit forms.
+ *
+ * @internal
+ */
+class AreaSummaryForm extends ContentEntityForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function prepareEntity() {
```

Comment:
Not sure if we need this?

Reply:
We don't actually need the entity forms at all so I removed them to simplify things